### ISSUE TITLE
Synctex not working with MiKTeX and Windows

### ISSUE THREAD

**Issue by tndrle** [tndrle/PyLuaTeX#8]
Based on https://tex.stackexchange.com/questions/631581/synctex-is-not-working-with-pyluatex-using-latexmk-in-texworks-miktex-windows

```
C:\Users\IEUser>python3.exe --version
Python 3.9.10
C:\Users\IEUser>lualatex --version
This is LuaHBTeX, Version 1.14.0 (MiKTeX 21.12)
```

Synctex is not working with MiKTeX under Windows 10 as soon as PyLuaTeX is included.

Synctex works:
```latex
\documentclass{article}
\begin{document}

Hello

\end{document}
```

Synctex does not work:
```latex
\documentclass{article}
\usepackage[executable=python.exe]{pyluatex}
\begin{document}

Hello

\end{document}
```

Log:
```
C:\Users\IEUser>lualatex -shell-escape -synctex=1 untitled-1
This is LuaHBTeX, Version 1.14.0 (MiKTeX 21.12)
 system commands enabled.
(./untitled-1.tex
LaTeX2e <2021-11-15> patch level 1
 L3 programming layer <2021-11-22>
(C:/Users/IEUser/AppData/Local/Programs/MiKTeX/tex/latex/base/article.cls
Document Class: article 2021/10/04 v1.4n Standard LaTeX document class
(C:/Users/IEUser/AppData/Local/Programs/MiKTeX/tex/latex/base/size10.clo))
(C:/Users/IEUser/AppData/Local/Programs/MiKTeX/tex/lualatex/pyluatex/pyluatex.s
ty (C:/Users/IEUser/AppData/Local/Programs/MiKTeX/tex/latex/l3kernel/expl3.sty
(C:/Users/IEUser/AppData/Local/Programs/MiKTeX/tex/latex/l3backend/l3backend-lu
atex.def))
(C:/Users/IEUser/AppData/Local/Programs/MiKTeX/tex/latex/kvoptions/kvoptions.st
y (C:/Users/IEUser/AppData/Local/Programs/MiKTeX/tex/latex/graphics/keyval.sty)
 (C:/Users/IEUser/AppData/Local/Programs/MiKTeX/tex/generic/ltxcmds/ltxcmds.sty
)
(C:/Users/IEUser/AppData/Local/Programs/MiKTeX/tex/generic/kvsetkeys/kvsetkeys.
sty))) (./untitled-1.aux)
(C:/Users/IEUser/AppData/Local/Programs/MiKTeX/tex/latex/base/ts1cmr.fd)
[1{C:/Users/IEUser/AppData/Local/MiKTeX/fonts/map/pdftex/pdftex.map}]
(./untitled-1.aux))
 406 words of node memory still in use:
   3 hlist, 1 vlist, 1 rule, 2 glue, 3 kern, 1 glyph, 4 attribute, 48 glue_spec
, 4 attribute_list, 1 write nodes
   avail lists: 2:23,3:4,4:1,5:22,6:2,7:38,9:18
<//?/c:/users/ieuser/appdata/local/programs/miktex/fonts/opentype/public/lm/lmr
oman10-regular.otf>
Output written on untitled-1.pdf (1 page, 3340 bytes).
SyncTeX: Can't rename untitled-1.synctex(busy) to untitled-1.synctex.gz
Transcript written on untitled-1.log.
lualatex: major issue: So far, you have not checked for MiKTeX updates.
```

Notice the line `SyncTeX: Can't rename untitled-1.synctex(busy) to untitled-1.synctex.gz`

**Comment by tndrle** [comment]
The problem exists under TeX Live, too (Windows 10)

**Comment by tndrle** [comment]
Fixed by release v0.4.1
